### PR TITLE
doc: instruction of enabling the laag secure boot

### PR DESCRIPTION
**Pull request by lirui34**
Add tutorial about how to enable the laag secure boot.

Signed-off-by: lirui34 <ruix.li@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
OK to verify

File: doc/tutorials/enable_laag_secure_boot.rst [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+   
+      acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge \
+        -s 2,pci-gvt -G "$3" \
+        -l com1,stdio \
+        -s 5,virtio-console,@pty:pty_port \
+        -s 6,virtio-hyper_dmabuf \
+        -s 3,virtio-blk,./clear-31080-kvm.img.signed \
+        -s 4,virtio-net,tap0 \
+        -s 7,virtio-rnd \
+        --ovmf w,/usr/share/acrn/bios/OVMF.fd \
```
Comment (peterfang) [inline doc/tutorials/enable_laag_secure_boot.rst]: After key enrollment, the "w" (writeback) option is optional. Users can leave it in the script if they want other EFI changes to be persistent. If the "w" option is removed, EFI will reset upon resets/shutdowns.

Comment (lirui34) [reply doc/tutorials/enable_laag_secure_boot.rst]: Thanks for the clarification, removed ``w,`` option in this section.

**Comment by peterfang** [comment]
Reviewed

**Pull request merged**